coderd/database/migrations/000052_workspace_app_health.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TYPE workspace_app_health AS ENUM ('disabled', 'initializing', 'healthy', 'unhealthy');
```

Comment:
`initializing` might be a bit misleading because it's really just paused waiting for the delay. What do you think about `waiting` instead?

Reply:
Or maybe `enabled` instead re the code checking if the URL is set below.

Reply:
It stays in initializing until it either gets healthy or unhealthy, which could take up to threshold * interval. I like initializing because it implies both enabled and waiting but no determined value yet.